### PR TITLE
[flutter_tool] Include impellerc output in ShaderCompilerException

### PR DESCRIPTION
**Pull request by zanderso**
This should hopefully give some better information in tool crash reports.

### CODE REVIEW

File: packages/flutter_tools/test/general.shard/build_system/targets/shader_compiler_test.dart [hunk 1]
```diff
@@ -130,13 +132,24 @@ void main() {
...
-      throwsA(isA<ShaderCompilerException>()),
-    );
+      );
+      fail('unreachable');
+    } on ShaderCompilerException catch (e) {
+      expect(e.toString(), contains('impellerc stdout:\nimpellerc stdout'));
+      expect(e.toString(), contains('impellerc stderr:\nimpellerc stderr'));
+    }
+
+    // await expectLater(
```
Comment (reddytocode) [inline packages/flutter_tools/test/general.shard/build_system/targets/shader_compiler_test.dart:146]: Do you need this commented code?

Comment (zanderso) [reply packages/flutter_tools/test/general.shard/build_system/targets/shader_compiler_test.dart]: Removed. Thanks!

**Comment by zanderso** [comment]
@christopherfujino friendly ping

**Comment by zanderso** [comment]
Related: https://github.com/flutter/flutter/issues/107590

**Review by christopherfujino — APPROVED** [review]
LGTM

**Pull request merged**